src/config.rs

```diff
@@ -334,7 +339,9 @@ macro_rules! build_cf_opt {
         cf_opts.set_soft_pending_compaction_bytes_limit($opt.soft_pending_compaction_bytes_limit.0);
         cf_opts.set_hard_pending_compaction_bytes_limit($opt.hard_pending_compaction_bytes_limit.0);
         cf_opts.set_optimize_filters_for_hits($opt.optimize_filters_for_hits);
-
+        if $opt.enable_doubly_skiplist {
+            cf_opts.set_doubly_skiplist();
```

Comment:
em, why not using the same style of other RocksDB functions? 

If I remember correctly, RocksDB will pass a bool var for this function.

Reply:
Config which is relation to memtable all use such style, like `rocksdb_options_set_doubly_skip_list_rep(rocksdb_options_t *opt)`,  [code](https://github.com/pingcap/rocksdb/blob/tikv-3.0/db/c.cc#2567).

Reply:
if I mean is `set_enable_double_skiplist(bool)`

like https://github.com/pingcap/rocksdb/blob/tikv-3.0/db/c.cc#L2451

Reply:
I get it. There has been default value for the `memtable_factory`, which is `SkipListFactory`, and it is defined in `options.h`.  So if we use `set_enable_double_skiplist(bool enable)`, we must know what kind of class the  `memtable_factory` should be when `enable` is set to be `false`.